packages/devtools_app/lib/src/info/flutter/info_screen.dart

```diff
@@ -44,20 +44,35 @@ class _InfoScreenBodyState extends State<InfoScreenBody> {
   FlutterVersion _flutterVersion;
 
   FlagList _flagList;
+  InfoController _controller;
 
   @override
   void initState() {
     super.initState();
-    InfoController(
-      onFlagListChanged: (flagList) => setState(() {
-        _flagList = flagList;
-      }),
-      onFlutterVersionChanged: (flutterVersion) => setState(() {
-        _flutterVersion = flutterVersion;
-      }),
+    _controller = InfoController(
```

Comment:
drive by fix for a bug where the controller wasn't properly disposed causing some asserts to throw.